README.md

```diff
@@ -698,13 +698,42 @@ showing the six most recent months in each:
         - SIT
         - UAT
         - PROD
+   
+   # Defect density  Segregated by Sprints
+   
+   Following are the configuraitons for creating defect report charts by Sprint.
```

Comment:
"configuraitons" is mis-spelled (here and in a few other places)

Reply:
I will fix that.